### PR TITLE
OpenNLX integration

### PR DESCRIPTION
**Pull request by mikbry**
Done #6

### CODE REVIEW

**Comment by willdurand** [comment]
`git pull --rebase origin master` + force push works better than using "update" on GitHub and it avoids these weird "merge branch 'master' into xxx" commits.

**Comment by mikbry** [comment]
I use github app and it works like that. So ok next time I will use command line to rebase

**Comment by willdurand** [comment]
Oh I did not know the GH app also had this feature, I wish github.com and this app would be more configurable 😕

**Comment by mikbry** [comment]
Context and complex response are not finished but we have basic behaviours

**Comment by mikbry** [comment]
I removed the [WIP] in title but WIP check stayed...

**Comment by willdurand** [comment]
that's because I have to review the PR first :D

**Comment by mikbry** [comment]
On another project, it displayed that the PR was waiting for review. Not here...

File: .travis.yml [hunk 1]
```diff
@@ -8,8 +8,8 @@ addons:
 
 cache: yarn
 script:
-  - bin/opla init --non-interactive
-  - mysql -e 'CREATE DATABASE opla_dev;'
-  - yarn db-migrate up
+  - mysql -e 'CREATE DATABASE opla_test;'
+  - bin/opla init --non-interactive --database-name opla_test
+  - bin/opla migrations up
```
Comment (willdurand) [inline .travis.yml]: mmh why is this part of the commit? Can you `git pull --rebase origin master` maybe?

**Pull request merged**